Sources/Sentry/Public/SentryUserFeedback.h

```diff
@@ -15,14 +15,24 @@ NS_ASSUME_NONNULL_BEGIN
  */
 NS_SWIFT_NAME(UserFeedback)
 @interface SentryUserFeedback : NSObject <SentrySerializable>
-SENTRY_NO_INIT
 
 /**
  * Initializes SentryUserFeedback and sets the required eventId.
  * @param eventId The eventId of the event to which the user feedback is associated.
+ * @note Uses the old envelope format descried at
+ * https://develop.sentry.dev/sdk/data-model/envelope-items/#user-feedback and will be deprecated in
+ * the future.
  */
 - (instancetype)initWithEventId:(SentryId *)eventId;
 
+/**
+ * Initializes a new `SentryUserFeedback` as its own event, instead of one attached to a transaction
```

Comment:
`l`: I think in the docs, we could refer to user feedback V0 and V1 or V1 and V2 instead of old and new. What if we again add a new version, is it then new new?

Reply:
A couple thoughts here. First, I agree from our earlier slack convo that using V1/V2 etc is bad practice, I've always disliked it, like with the earlier PerformanceV2 stuff. So I don't know why we'd want to do it here now.

Second, if we are already rolling out a new new user feedback paradigm (V3) on top of the old new one (V2) before even finishing deprecating/removing the original (V1), we need to step back and take a breath 😆 

We are going to roll the manual calls to capture feedback directly and/or from a custom built UI into this new paradigm, by extending the current functionality (report feedback with an attached event ID) to not _require_ the event ID, and also be able to attach screenshots. And then provide the managed UI to capture the inputs that go into the same callpath.

Do we really anticipate a completely different thing coming any time soon?

What we can do here is simply remove "old" and "new" and add a deprecation message at the appropriate time. They don't have to be "old" vs "new", they can just be "different", and "deprecated" vs "supported".

Reply:
Oh and also, I'm going to put back SentryUserFeedback and just use the new type SentryFeedback that I introduced for this separate pathway.